src/SpotifyApi.NetCore/FollowApi.cs

```diff
@@ -134,14 +134,193 @@ public FollowApi(HttpClient httpClient) : base(httpClient)
             string accessToken = null
             )
         {
-            if (playlistId?.Length < 1) throw new ArgumentNullException("playlistId");
+            if (playlistId?.Length < 1) throw new 
```

Comment:
For this validation check the `ArgumentNullException` is correct. I prefer `string.IsNullOrEmpty()` (it is more efficient and easier to read). It may be a minor thing, but you should avoid counting the length of a string unless you absolutely have to 🙂 e.g.

    if (string.IsNullOrEmpty(playlistId)) throw new ArgumentNullException("playlistId");